utils/qemu/patches/0010-no-tests.patch

```diff
@@ -1,6 +1,6 @@
 --- a/meson.build
 +++ b/meson.build
```

Comment:
I see that this patch was created by @neheb. Would be good if there is some argument in meson to disable such tests, though.